docs/developer/getting-started/debugging.asciidoc

```diff
@@ -56,4 +56,6 @@ ELASTIC_APM_ACTIVE=true yarn start
 Once the agent is active, it will trace all incoming HTTP requests to
 {kib}, monitor for errors, and collect process-level metrics. The
 collected data will be sent to the APM Server and is viewable in the APM
-UI in {kib}.
\ No newline at end of file
+UI in {kib}.
+
+If you want to run kibana with a local instance of the APM agent, refer to <<running-kibana-with-apm>>.
```

Comment:
Self review: I've linked to the documentation on how to run Kibana with a local APM agent here.